src/TrueLayer/Mandates/MandatesApi.cs

```diff
@@ -52,5 +54,48 @@ public async Task<ApiResponse<CreateMandateResponse>> CreateMandate(CreateMandat
                 cancellationToken
             );
         }
+
+        /// <inheritdoc />
+        public async Task<ApiResponse<MandateDetailUnion>> GetMandate(string mandateId, MandateType mandateType, CancellationToken cancellationToken = default)
+        {
+            mandateId.NotNullOrWhiteSpace(nameof(mandateId));
+
+            ApiResponse<GetAuthTokenResponse> authResponse = await _auth.GetAuthToken(new GetAuthTokenRequest($"recurring_payments:{mandateType}"), cancellationToken);
+
+            if (!authResponse.IsSuccessful)
+            {
+                return new(authResponse.StatusCode, authResponse.TraceId);
+            }
+
+            return await _apiClient.GetAsync<MandateDetailUnion>(
+                new Uri(_baseUri, $"/v3/mandates/{mandateId}"),
+                authResponse.Data!.AccessToken,
+                cancellationToken
+            );
+        }
+
+        /// <inheritdoc />
+        public async Task<ApiResponse<ResourceCollection<MandateDetailUnion>>> ListMandates(ListMandatesQuery query, MandateType mandateType, CancellationToken cancellationToken = default)
+        {
+            ApiResponse<GetAuthTokenResponse> authResponse = await _auth.GetAuthToken(new GetAuthTokenRequest($"recurring_payments:{mandateType}"), cancellationToken);
+
+            if (!authResponse.IsSuccessful)
+            {
+                return new(authResponse.StatusCode, authResponse.TraceId);
+            }
+
+            var queryParameters = System.Web.HttpUtility.ParseQueryString(string.Empty);
+            queryParameters["client_id"] = _options.ClientId;
```

Comment:
Why `client_id` is here?

I don't see it [mentioned in the docs](https://docs.truelayer.com/reference/list-mandate).